### PR TITLE
Create es.json

### PR DESCRIPTION
**Pull request by Blacker2911**
This a request to create es.json

### CODE REVIEW

File: resources/translations/es.json [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+  "GUILD_COUNT": "Conteo del servidor",
+  "HELP_COMMAND": "Ayuda",
+  "HELP_COMMAND_INFO": "Para más información acerca del comando escribe `/help command [nombre del comando]`",
+  "HELP_COMMAND_INFO_LONG": "Para más información acerca del comando escribe `/help command [command name]`. \nEscribe `/help all` para ver la lista de comandos.",
+  "HELP_COMMAND_NAME_FIND": "Comando con el nombre `{{name}}` no encontrado",
+  "HELP_COMMAND_NAME_INVALID": "Nombre de comando inválido",
+  "HELP_COMMAND_NAME_TITLE": "Ayuda por {{name}}:",
+  "HELP_DELETE_DATA": "¿Quieres realizar una solicitud para eliminar tu información de usuario?",
+  "HELP_DELETE_DATA_DETAIL": "En el servidor anexado, ve al canal #data-deletion-request. Dentro del canal, escribe /deletedata.",
+  "HELP_DEVELOPER": "Este es un comando para **desarrollador**,
```
Comment (coolcalcacol) [inline resources/translations/es.json]: String not closed

File: resources/translations/es.json [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+  "HELP_DEVELOPER_DETAIL": "Sólo los desarrolladores del bot pueden usar este comando.",
+  "HELP_INVITE": "¿Quieres usar el bot en tu servidor?",
+  "HELP_INVITE_LINK": "[Invitación al servidor Private6Mans({{link}})",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Square bracket not closed

File: resources/translations/es.json [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+  "RESET_MATCHES": "Partidas restablecidas",
+  "RESET_MATCH_RESULTS": "Restablecer resultados de la partida",
+  "RESET_MODE": "Por favor especifica un modo",
+  "RESET_PLAYERS": "Jugadores restablecidos",
+  "RESET_QUEUES": "Lista de espera restablecids",
+  "RESET_ROOMS": "Por favor especifica las salas a reiniciar",
+  "RESET_STATS": "Reiniciar estadísticas",
+  "RESET_SUCCESS": "Reinicio del servidor0",
+  "REST_LATENCY": "REST Latency Reiniciar latencia",
+  "RE_QUEUED": "YHas reingresado a la lista de espera.",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Is this a typo or intended?

File: resources/translations/es.json [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+  "REPORT_PLAYER_DESCRIPTION": "¿Ganaste o perdiste esta partida?",
+  "RESET_CONFIRMATION": "¿Estás seguro que quieres eliminar tu información? Esta acción no se puede deshacer posteriormente ",
+  "RESET_MATCHES": "Partidas restablecidas",
+  "RESET_MATCH_RESULTS": "Restablecer resultados de la partida",
+  "RESET_MODE": "Por favor especifica un modo",
+  "RESET_PLAYERS": "Jugadores restablecidos",
+  "RESET_QUEUES": "Lista de espera restablecids",
+  "RESET_ROOMS": "Por favor especifica las salas a reiniciar",
+  "RESET_STATS": "Reiniciar estadísticas",
+  "RESET_SUCCESS": "Reinicio del servidor0",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Typo

File: resources/translations/es.json [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+  "PROCESS_MEMORY_USAGE": "Procesar uso de memoria",
+  "QUEUE_CLEARED": "Lista de espera limpia!",
+  "QUEUE_EMPTY": "La lista de espera está vacía",
+  "QUEUE_JOIN": "{{player}} se ha unido a la lista de espera!",
+  "QUEUE_LEAVE": "{{player}} ha dejado la lista de espera!",
+  "QUEUE_START": "{{player}} ha iniciado una lista de espera!",
+  "RATING": "Clasificar",
+  "REPORT_MOD_DESCRIPTION": "¿Qué equipo ganó esta partida?",
+  "REPORT_PLAYER_DESCRIPTION": "¿Ganaste o perdiste esta partida?",
+  "RESET_CONFIRMATION": "¿Estás seguro que quieres eliminar tu información? Esta acción no se puede deshacer posteriormente ",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Extra space

File: resources/translations/es.json [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+  "HELP_DEVELOPER_DETAIL": "Sólo los desarrolladores del bot pueden usar este comando.",
+  "HELP_INVITE": "¿Quieres usar el bot en tu servidor?",
+  "HELP_INVITE_LINK": "[Invitación al servidor Private6Mans({{link}})",
+  "HELP_ISSUES": "¿Tienes problema?",
+  "HELP_ISSUES_LONG": "Asegúrate de que el bot tenga permisos de leer y enviar mensajes, enviar links y agregar reacciones. Además, asegúrate de que todos en la sala puedan recibir mensajes privados del bot.",
+  "HELP_LIST_DEVELOPER": "Lista de comandos del desarrollador",
+  "HELP_LIST_MODERATOR": "Lista de comandos del moderador",
+  "HELP_LIST_PUBLIC": "Lista de comandos públicos",
+  "HELP_MODERATOR": "Este es un comando de **moderador**",
+  "HELP_MODERATOR_DETAIL": "Sólo usuarios con el rol de moderador designado por Private 6 Mans (/config modrole), o usuarios con los permisos de creador y administrador del servidor pueden usar este comando.",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: @TheBeardedB this isn't the command for mod role anymore is it? Something for me to fix in my English doc too?

File: resources/translations/es.json [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+  "GENERIC_STOP": "Detener este menú",
+  "GENERIC_TEAM": "Equipo {{number}}",
+  "GENERIC_TIME": "Hora",
+  "GENERIC_WIN": "Victoria",
+  "GENERIC_WINNER": "Ganador",
+  "GETTING_PING": "Obteniendo Ping!",
+  "GUILD_COUNT": "Conteo del servidor",
+  "HELP_COMMAND": "Ayuda",
+  "HELP_COMMAND_INFO": "Para más información acerca del comando escribe `/help command [nombre del comando]`",
+  "HELP_COMMAND_INFO_LONG": "Para más información acerca del comando escribe `/help command [command name]`. \nEscribe `/help all` para ver la lista de comandos.",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: [command name] still in English unlike above

File: resources/translations/es.json [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+  "ERROR_NO_MATCH_RESULT": "Ningún resultado ingresado, por favor digita alguno.",
+  "ERROR_NO_REPORT": "No puedes reportar una partida a la que no perteneces.",
+  "ERROR_PERMISSIONS": "Me faltan permisos!",
+  "ERROR_REQUIRED_ROLE": "No tienes el rol requerido para ingresar a la lista de espera!",
+  "ERROR_RESPONSE": "Respuesta inválida",
+  "ERROR_ROLE": "Rol inválido",
+  "ERROR_SET": "No configurado",
+  "ERROR_TIME": "Se te acabó el tiempo!",
+  "ERROR_UNDO": "Sólo puedes deshacer una artida que ha sido reportada o cancelada",
+  "ERROR_UNKNOWN_DESCRIPTION": "Escribe este código en el canal #error-report en el servidor de 6 Mans para que podamos examinarlo. ",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Whitespace

File: resources/translations/es.json [hunk 9]
```diff
@@ -0,0 +1,231 @@
...
+  "DELETE_DATA_CONFIRMATION": "¿Estás seguro que quieres eliminar tu información? Esta acción no se puede deshacer posteriormente.",
+  "DELETE_DATA_DETAIL": "Tu información fue eliminada de nuestra base de datos.",
+  "DELETE_DATA_SUCCESS": "Información eliminada",
+  "DELETE_DATA_TITLE": "Eliminar información",
+  "EMBED_CAPTAINS_FIELD_NAME": "Elección de equipos.",
+  "EMBED_CAPTAINS_FIELD_VALUE": "Ahora los capitanes elegirán los miembros de su equipo!",
+  "EMPTY_BLACKLIST": "Ningún jugador en la lista de bloqueados",
+  "ERROR_ALREADY_QUEUED": "¡Ya estás en la lista de espera!",
+  "ERROR_CANCELLED": "Operación cancelada!",
+  "ERROR_CAPTAIN_DM": "No puedo enviar mensaje privado a {{captain}}, ¿tienes activada esta opción? (Elección de equipos cancelada)  ",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Whitespace

File: resources/translations/es.json [hunk 10]
```diff
@@ -0,0 +1,231 @@
...
+  "CREATE_MATCH_EMBED_TITLE": "Elección de Equipo",
+  "CREATE_MATCH_ID": "ID de la partida (Usa esto para reportar el marcador)",
+  "CREATE_MATCH_INVALID_SELECTION": "Selección inválida, por favor elige nuevamente",
+  "CREATE_MATCH_NOW_PICKING": "{{captain}} está eligiendo",
+  "CREATE_MATCH_PLAYER_CHOSEN_COMPLETE": "{{user}} elegido! Selección de Equipos Completa.",
+  "CREATE_MATCH_PLAYER_CHOSEN_ONGOING": "{{user}} elegido! ({{count}}/{{total}})",
+  "CREATE_MATCH_RANDOM": "Equipos aleatorios",
+  "CREATE_MATCH_RETURN_DESCRIPTION": "La votación ha finalizado. Los resultados son:",
+  "CREATE_MATCH_RETURN_TITLE": "Votación completa!",
+  "CREATE_MATCH_STARTER_DESCRIPTION": "Este mensaje necesita tener **{{count}}** interacciones de jugadores para continuar!  ",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: Whitespace

**Review by coolcalcacol — CHANGES_REQUESTED** [review]
Couple of small points after a quick review

File: resources/translations/es.json (hunk 6 above)
Comment (TheBeardedB) [reply resources/translations/es.json]: it's now /config guild mod-role

File: resources/translations/es.json [hunk 11]
```diff
@@ -135,16 +135,16 @@
...
   "HELP_COMMAND_NAME_INVALID": "Nombre de comando inválido",
   "HELP_COMMAND_NAME_TITLE": "Ayuda por {{name}}:",
   "HELP_DELETE_DATA": "¿Quieres realizar una solicitud para eliminar tu información de usuario?",
   "HELP_DELETE_DATA_DETAIL": "En el servidor anexado, ve al canal #data-deletion-request. Dentro del canal, escribe /deletedata.",
-  "HELP_DEVELOPER": "Este es un comando para **desarrollador**,
+  "HELP_DEVELOPER": "Este es un comando para **desarrollador**",
   "HELP_DEVELOPER_DETAIL": "Sólo los desarrolladores del bot pueden usar este comando.",
   "HELP_INVITE": "¿Quieres usar el bot en tu servidor?",
-  "HELP_INVITE_LINK": "[Invitación al servidor Private6Mans({{link}})",
+  "HELP_INVITE_LINK": "[Invitación al servidor Private6Mans({{link}})]",
```
Comment (coolcalcacol) [inline resources/translations/es.json]: `"[Invitación al servidor Private6Mans]({{link}})",`

**Review by coolcalcacol — CHANGES_REQUESTED** [review]
One more thing + the change to /config guild modrole

**Review by coolcalcacol — APPROVED** [review]

File: resources/translations/es.json [hunk 12]
```diff
@@ -0,0 +1,231 @@
...
+  "ERROR_NO_REPORT": "No puedes reportar una partida a la que no perteneces.",
+  "ERROR_PERMISSIONS": "Me faltan permisos!",
+  "ERROR_REQUIRED_ROLE": "No tienes el rol requerido para ingresar a la lista de espera!",
+  "ERROR_RESPONSE": "Respuesta inválida",
+  "ERROR_ROLE": "Rol inválido",
+  "ERROR_SET": "No configurado",
+  "ERROR_TIME": "Se te acabó el tiempo!",
+  "ERROR_UNDO": "Sólo puedes deshacer una artida que ha sido reportada o cancelada",
+  "ERROR_UNKNOWN_DESCRIPTION": "Escribe este código en el canal #error-report en el servidor de 6 Mans para que podamos examinarlo.", 
+  "ERROR_UNKNOWN_ID": " ID del error: {{error}}"
```
Comment (TheBeardedB) [inline resources/translations/es.json]: Missing comma ',' on the end of the line.

**Pull request merged**